code/modules/surgery/healing.dm

```diff
@@ -72,6 +71,7 @@
 		urhealedamt_burn *= 0.55
 		umsg += " as best as you can while they have clothing on"
 		tmsg += " as best as they can while [target] has clothing on"
+	experience_given = CEILING(((urhealedamt_burn+urhealedamt_burn)/MEDICAL_SKILL_EASY),1)
 	target.heal_bodypart_damage(urhealedamt_brute,urhealedamt_burn)
```

Comment:
```suggestion
	var/healeddammix = target.heal_bodypart_damage(urhealedamt_brute,urhealedamt_burn)
	experience_given = CEILING(((healeddammix)/MEDICAL_SKILL_EASY),1)
```
If you want xp gain based on REAL hp healed you need use this.
https://github.com/tgstation/tgstation/blob/024d4d85818eebf364fdbd274200a6bd786b641f/code/modules/mob/living/damage_procs.dm#L256-L257
https://github.com/tgstation/tgstation/blob/024d4d85818eebf364fdbd274200a6bd786b641f/code/modules/mob/living/damage_procs.dm#L157-L163
https://github.com/tgstation/tgstation/blob/024d4d85818eebf364fdbd274200a6bd786b641f/code/modules/mob/living/damage_procs.dm#L206-L212